package.json

```diff
@@ -54,7 +54,7 @@
     "@types/eslint": "=7.2.10",
     "@types/estree": "0.0.52",
     "@types/node": "^4.9.5",
-    "@typescript-eslint/parser": "^2.34.0 || ^3.10.1 || ^4.0.0 || ^5.0.0",
+    "@typescript-eslint/parser": "^2.34.0 || ^3.10.1 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0",
```

Comment:
```suggestion
    "@typescript-eslint/parser": "^2.34.0 || ^3.10.1 || ^4 || ^5 || ^6.20 || ^7.14.1 || ^8.4",
```

Reply:
I get an issue with Node v19 here, it only allows installing up to v7.2.0 since then the engines field is introduced which doesn't list 19 🙂

Reply:
let's tweak the workflow then, so it installs v6 < 20, and v7 on node 20, and v8 on node 22?

Reply:
Disabled v7 on Node 19 which was enough to get [all steps green](https://github.com/hampustagerud/eslint-plugin-react/actions/runs/10745877419) 👍

Reply:
@hampustagerud i'm not sure i see this commit pushed up to your branch - either my suggested change, or anything about v7 on node 19

Reply:
I added it to the commits already in the branch

- [fa6c54007f24131c02d65f725c566c4228da6b66](https://github.com/hampustagerud/eslint-plugin-react/commit/fa6c54007f24131c02d65f725c566c4228da6b66)
- [9bdbcd1931df5787e863e1ccdd4cd1a85bbc88a4](https://github.com/hampustagerud/eslint-plugin-react/commit/9bdbcd1931df5787e863e1ccdd4cd1a85bbc88a4) 

If you re-pick my commits it should show up, or I can reset to what was there and add it in new commits if that's easier 🙂